chapter_02/lewis/exercise_3.pl

```diff
@@ -0,0 +1,28 @@
+###
+# (C) 2017 Lewis Maitland
+# O'Reily Excersises
+###
+
+use strict;
+use warnings;
+
+use constant TITLE => " MULTIPLICATOR v1.01 ";

```

Comment:
Probably don't need to use constants in scripts this simple. You're not the first java developer I've seen do this though.